code/modules/species/station/xenomorph_hybrids/hybrid_resin.dm

```diff
@@ -190,4 +190,3 @@
 	density = 0
 	plane = TURF_PLANE
 	layer = ABOVE_TURF_LAYER
-	color = "#422649"
```

Comment:
everything looks fine, just double checking: was this an intended change

Reply:
yes, the sprite is already coloured, which means giving more color makes it completely black